code/modules/client/preferences.dm

```diff
@@ -350,31 +382,37 @@ var/global/list/special_role_times = list( //minimum age (in days) for accounts
 				var/headaccessoryname = "Head Accessory"
 				if(species == "Unathi")
 					headaccessoryname = "Horns"
 				dat += "<br><b>[headaccessoryname]</b><br>"
 				dat += "<a href='?_src_=prefs;preference=headaccessory;task=input'>Change Color</a> <font face='fixedsys' size='3' color='#[num2hex(r_headacc, 2)][num2hex(g_headacc, 2)][num2hex(b_headacc, 2)]'><table style='display:inline;' bgcolor='#[num2hex(r_headacc, 2)][num2hex(g_headacc, 2)][num2hex(b_headacc)]'><tr><td>__</td></tr></table></font> "
 				dat += "Style: <a href='?_src_=prefs;preference=ha_style;task=input'>[ha_style]</a><br>"
-
-				dat += "<br><b>Body Markings</b><br>"
+			if(species in list("Unathi", "Vulpkanin", "Tajaran", "Machine"))
+				var/markings = "Body Markings"
+				if(species == "Machine")
+					markings = "Optic Modification"
+				dat += "<br><b>[markings]</b><br>"
 				dat += "<a href='?_src_=prefs;preference=markings;task=input'>Change Color</a> <font face='fixedsys' size='3' color='#[num2hex(r_markings, 2)][num2hex(g_markings, 2)][num2hex(b_markings, 2)]'><table style='display:inline;' bgcolor='#[num2hex(r_markings, 2)][num2hex(g_markings, 2)][num2hex(b_markings)]'><tr><td>__</td></tr></table></font> "
 				dat += "<br>Style: <a href='?_src_=prefs;preference=m_style;task=input'>[m_style]</a><br>"
 
 			var/hairname = "Hair"
 			if(species == "Machine")
-				hairname = "Frame Color"
+				hairname = "Wig/Screen Display"
 			dat += "<br><b>[hairname]</b><br>"
 			dat += "<a href='?_src_=prefs;preference=hair;task=input'>Change Color</a> <font face='fixedsys' size='3' color='#[num2hex(r_hair, 2)][num2hex(g_hair, 2)][num2hex(b_hair, 2)]'><table style='display:inline;' bgcolor='#[num2hex(r_hair, 2)][num2hex(g_hair, 2)][num2hex(b_hair)]'><tr><td>__</td></tr></table></font> "
 			dat += " <br>Style: <a href='?_src_=prefs;preference=h_style;task=input'>[h_style]</a><br>"
 
-			dat += "<br><b>Facial</b><br>"
+			if(species == "Machine")
+				hairname = "Postiche"
+			dat += "<br><b>[hairname]</b><br>"
 			dat += "<a href='?_src_=prefs;preference=facial;task=input'>Change Color</a> <font face='fixedsys' size='3' color='#[num2hex(r_facial, 2)][num2hex(g_facial, 2)][num2hex(b_facial, 2)]'><table  style='display:inline;' bgcolor='#[num2hex(r_facial, 2)][num2hex(g_facial, 2)][num2hex(b_facial)]'><tr><td>__</td></tr></table></font> "
-			dat += " <br>Style: <a href='?_src_=prefs;preference=f_style;task=input'>[f_style]</a><br>"
+			dat += " <br>Style: <a href='?_src_=prefs;preference=f_style;task=input'>[f_style ? "[f_style]" : "Shaved"]</a><br>"
 
-			dat += "<br><b>Eyes</b><br>"
-			dat += "<a href='?_src_=prefs;preference=eyes;task=input'>Change Color</a> <font face='fixedsys' size='3' color='#[num2hex(r_eyes, 2)][num2hex(g_eyes, 2)][num2hex(b_eyes, 2)]'><table  style='display:inline;' bgcolor='#[num2hex(r_eyes, 2)][num2hex(g_eyes, 2)][num2hex(b_eyes)]'><tr><td>__</td></tr></table></font><br>"
+			if(!(species == "Machine"))

```

Comment:
Eye colour doesn't affect IPCs exclusively, their face is covered by their 'hair' (screens) if they are using the Morpheus heads

If they are not using the default Morpheus head, they have the option to choose an 'optics' marking (I chose body markings for this optic because it was closest to the layer eyes naturally get rendered on) and that still doesn't use eye colour, it uses body marking colour

I didn't see a reason for the eyes section to be listed for IPCs if it does nothing

The reason why head accessory is still listed is because it's going to be used for the antennae suggestion